app/models/article_translation.rb

```diff
@@ -1,12 +1,14 @@
 class ArticleTranslation < ApplicationRecord
+  include ArticleData
```

Comment:
It could be called ArticleStatuses actually, cause there is no data here, there are just constants for status.